transcom/mymove
# Add PPM Type and PPM Estimated Weight to MTO after counseling

## Description

Update an MTO with a PPM's type and estimated weight on the post-counseling endpoint

## Setup
`make db_dev_migrate`
`make server_test`

To test on postman:
- `make db_dev_e2e_populate`
- Get all move task orders
- copy ID of one of the MTOs
- `{{baseUrl}}/move-task-orders/:moveTaskOrderID/post-counseling-info`
- Add `If-Match` header `2020-02-27T16:09:02.187Z`
- Add body `{ "ppm_type": "FULL", "ppm_estimated_weight": 4000 }
Send request and the response should have updated ppm_type and ppm_estimated_weight in payload

## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/MB-641) for this change
